src/services/createRoute.spec-d.ts

```diff
@@ -41,7 +40,7 @@ test('options with path', () => {
   type Source = typeof route['path']
   type Expect = { value: '/foo', params: {} }
 
-  expectTypeOf<Source>().toEqualTypeOf<Expect>()
+  expectTypeOf<Source>().toMatchTypeOf<Expect>()
```

Comment:
I switched these to `withParams` because the `WithParams` type also has the `toString: () => string` and I didn't want to include that in each of these `Expect` cases. It still tests what I think we're trying to assert